### PR TITLE
Vet Center v3 components added, built, updated

### PR DESCRIPTION
**Pull request by eselkin**
## Summary

- Adds va-links where possible (not possible in contents of accordions for the most part because those are controlled by CMS editors - we have to inform them to create links in text bodies with the `<va-link href="" text="">` component). Updated to v3 accordions and items. Updated nested content paragraph items to use v3 (like alerts in accordions). Replaced vets-website operating status with generalized facility liquid template with va-alert-expandable (should merge VBA one and this one). Updated all telephone links (when not brought in as an `<a href="tel:...">` tag from drupal) to use va-telephone. Used liquid filter to separate phone numbers and extension when unprocessed phone numbers are received.
- Sitewide Facilities

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16945

## Testing done

- Checked old site and uploaded screenshots. Added testing for new features such as separating phone number and extension. Modified testing and liquid filters that checked for `<a>` tags created around phone numbers to use `<va-telephone>` components.
- Tested on tugboat and verified new testing succeeds.
- _Describe the tests completed and the results_
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

<details>
<summary>Before Vet Center Main Page (Desktop)</summary>

</details>

<details>
<summary>Current Vet Center Main Page (Desktop)</summary>

</details>

<details>
<summary>Before Vet Center Main Page (Mobile)</summary>

</details>

<details>
<summary>Current Vet Center Main Page (Mobile)</summary>

</details>

<details>
<summary>Current Vet Center Locations Page (Desktop)</summary>

</details>

<details>
<summary>Current Vet Center Locations Page (Mobile)</summary>

</details>

## What areas of the site does it impact?

Changes the modernized Vet Center pages to use DS v3 components when possible and v1 components when available. 

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [x] Documentation has been updated ([link to documentation](#) \*if necessary)
- [x] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [x] Events are being sent to the appropriate logging solution
- [x] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [x] Did you login to a local build and verify all authenticated routes work as expected with a test user

## Requested Feedback

View pages:
This pulls up all the changes:
https://web-ihqrv6rby5lxlkn8zqdxf9u7lk1aplcp.demo.cms.va.gov/boston-vet-center/
The locations page only partially works on tugboats because mapbox doesn't work there:
https://web-ihqrv6rby5lxlkn8zqdxf9u7lk1aplcp.demo.cms.va.gov//boston-vet-center/locations/

### CODE REVIEW

**Comment by jilladams** [comment]
@eselkin @maxx1128 is this PR blocked from code review on CI stuff?

File: src/site/includes/operating_status_expandable_alert.liquid [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  {% endcase %}
+  <div class="vads-u-margin-bottom--1">
+    <va-alert-expandable
+      id="operating-status-{{ fieldFacilityLocatorApiId }}"
+      trigger="{{ trigger }}"
+      status="{{ status }}"
+      class="vads-u-margin-y--0 vamc-facility-expandable-alert"
+      style="max-width: 303px;"
+    >
+      <div>
```
Comment (maxx1128) [inline src/site/includes/operating_status_expandable_alert.liquid:21]: **Question:** is this extra `div` wrapper needed here? It doesn't look like it has any semantic or styling purpose.

**Review by maxx1128 — CHANGES_REQUESTED** [review]

File: src/site/layouts/vet_center.drupal.liquid [hunk 2]
```diff
@@ -105,7 +100,8 @@
                       Direct line</h4>
                     <div class="vads-u-margin-bottom--3">
                       <div class="main-phone vads-u-margin-bottom--1">
-                        </strong><a href="tel:{{ fieldPhoneNumber }}">{{ fieldPhoneNumber }}</a>
+                        {% assign phoneNumber = fieldPhoneNumber | separatePhoneNumberExtension %}
+                        </strong><va-telephone contact="{{phoneNumber.phoneNumber}}" extension="{{phoneNumber.extension}}"></va-telephone>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/layouts/vet_center.drupal.liquid:104]: I think they had it as <strong /> before, but equally pointless. Will remove.

File: src/site/includes/operating_status_expandable_alert.liquid (hunk 1 above)
Comment (eselkin) [reply src/site/includes/operating_status_expandable_alert.liquid:21]: Removed

File: src/site/layouts/vet_center.drupal.liquid (hunk 2 above)
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/layouts/vet_center.drupal.liquid]: removed

File: src/site/includes/vet_centers/address_phone_image.liquid [hunk 3]
```diff
@@ -63,14 +63,13 @@
...
-            </div>
+            {% assign mainPhone = mainVetCenterPhone | separatePhoneNumberExtension %}
           {% else %}
-            <div class="main-phone">
-              </strong><a href="tel:{{ vetCenter.fieldPhoneNumber }}">{{ vetCenter.fieldPhoneNumber }}</a>
-            </div>
+            {% assign mainPhone = vetCenter.fieldPhoneNumber | separatePhoneNumberExtension %}
           {% endif %}
+          <div class="main-phone">
+            </strong><va-telephone contact="{{mainPhone.phoneNumber}}" extension="{{mainPhone.extension}}"></va-telephone>
```
[the remark this answers is not in the document]
Comment (eselkin) [reply src/site/includes/vet_centers/address_phone_image.liquid]: changed

**Review by maxx1128 — APPROVED** [review]
👍

**Review by chriskim2311 — APPROVED** [review]
LGTM!

**Comment by laflannery** [comment]
@eselkin I took a look at this and found a couple things:
- The expandable alert is not showing up on the main vet center page. [Harlem Vet center](http://9e11b355dbb7bbb39750e86ce0cd6b12.review.vetsgov-internal/harlem-vet-center/) is closed and has the proper status set. When you first go to the page you see the alert for about 1 or 2 seconds and then it disappears.
  - The alerts are working correctly in [the Nearby locations section](http://9e11b355dbb7bbb39750e86ce0cd6b12.review.vetsgov-internal/harlem-vet-center/locations/) though
- However, in the nearby locations section, the Titles of the facilities no longer link. Can we relink these names using the `<va-link>` component? PR on the left, Prod on the right:
image

I also have one question - there are no vet centers with exts in their phone number so I just want to make sure. Will all phone numbers within the components all still display properly if an ext is added in the future?

**Comment by laflannery** [comment]
@eselkin Alerts look good now, thanks!

**Review by jamigibbs — APPROVED** [review]

**Pull request merged**